airbyte-integrations/connectors/source-exchange-rates-tutorial/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,22 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "customers",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh"]
+      },
+      "sync_mode": "full_refresh",
+      "destination_sync_mode": "overwrite"
+    },
+    {
+      "stream": {
+        "name": "employees",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh", "incremental"]
+      },
+      "sync_mode": "incremental",
```

Comment:
I guess we should provide a cursor field in both the stream and the configured_stream in case we set the incremental mode here